docs/pages/example/mapbox-gl-geocoder-local-geocoder.html

```diff
@@ -0,0 +1,64 @@
+<script src='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v2.2.0/mapbox-gl-geocoder.min.js'></script>
+<link rel='stylesheet' href='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v2.2.0/mapbox-gl-geocoder.css' type='text/css' />
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v9',
+    center: [-79.4512, 43.6568],
+    zoom: 13
+});
+
+/* given a query returns a matching geographic coordinates as search results in
+ * carmen geojson format, https://github.com/mapbox/carmen/blob/master/carmen-geojson.md
+ */
+var coordinatesGeocoder = function (query) {
+    // match anything which looks like a decimal degrees coordinate pair
+    var matches = query.match(/^[ ]*(-?\d+\.?\d*)[, ]+(-?\d+\.?\d*)[ ]*$/);
+    if (!matches) {
+        return null;
+    }
+
+    function coordinateFeature(lng, lat) {
+        return {
+            center: [lng, lat],
+            geometry: {
+                type: "Point",
+                coordinates: [lng, lat]
+            },
+            place_name: 'Lat: ' + lat + ', Lng: ' + lng, // eslint-disable-line camelcase
```

Comment:
yes but I don't see any other way around it since `place_name` is defined in the [carmen geojson spec](https://github.com/mapbox/carmen/blob/master/carmen-geojson.md)